lib/ClangImporter/ClangImporter.cpp

```diff
@@ -319,11 +320,13 @@ class BridgingPPTracker : public clang::PPCallbacks {
 class ClangImporterDependencyCollector : public clang::DependencyCollector
 {
   llvm::StringSet<> ExcludedPaths;
+  std::shared_ptr<llvm::FileCollector> FileCollector;
```

Comment:
Can you leave a comment here explaining that this is for LLDB to generate reproducers and does not represent a precise view of a module's dependencies?